src/renderer/qt/qt_window_manager.cc

```diff
@@ -341,16 +341,61 @@ void FillCandidates(const commands::Candidates &candidates,
 }
 }  // namespace
 
+struct VirtualRect {
+  const QScreen *screen;
+  const Rect rect;
+
+  VirtualRect(const QScreen *screen, const Rect rect)
+      : screen(screen), rect(rect) {}
+};
+
+Rect TranslateToVirtualRectWithScreen(const QScreen *screen,
```

Comment:
I find the function name a bit confusing because it returns `Rect`, not `VirtualRect`.

Reply:
I removed  VirtualRectWithScreen from the fuction